test/api/fragments.test.js

```diff
@@ -25,7 +27,7 @@ const correctRequest = {
 jest.unmock('api/fragments');
 jest.mock('@entando/apimanager', () => ({
   makeRequest: jest.fn(() => new Promise(resolve => resolve({}))),
-  METHODS: { GET: 'GET', DELETE: 'DELETE' },
+  METHODS: { GET: 'GET', DELETE: 'DELETE', POST: 'POST' },
```

Comment:
`@entando/apimanager` is mocked globally, you can remove this whole `jest.mock()` block